src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, auto);
+  justify-content: center;
+  padding: 50px;
+  column-gap: 48px;
+  row-gap: 46px;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
```

Comment:
It would be great to create variables for all main values (main sizes, colors etc)